docs/UserGuide.adoc

```diff
@@ -272,11 +272,23 @@ Examples:
 ==== Saving a todo eatery to the main eatery list : `save`
 
 Remove a todo eatery from the todo list and provides a quick way to add it to the main list of eateries. +
-Format: `save [id]` (saves the eatery to the main list)
+
+****
+* Save the eatery at the specified index. The index refers to the index number shown in the displayed todo list.
+* The index *must be a positive integer* 1, 2, 3, ...
+* The eatery will be removed from the todo list after `save` is executed
+* User will be required to input the missing field before the eatery is saved in the main list.
+****
+
+Format: `save [index]` (saves the eatery to the main list)
```

Comment:
I think if u still wna do the bullet points then it should be like this:

Remove a todo eatery from the todo list and provides a quick way to add it to the main list of eateries. 
Format: `save [index]` 
****
* Save the eatery at the specified index. The index refers to the index number shown in the displayed todo list.
* The index *must be a positive integer* 1, 2, 3, ...
* The eatery will be removed from the todo list after `save` is executed
* User will be required to input the missing field before the eatery is saved in the main list.
****

Reply:
You are referring to the format: save 1 being above the bullet point is it?